pkg/common/common.go

```diff
@@ -0,0 +1,60 @@
+package common
```

Comment:
We should name this package appropriately.. using generic names will take away the readability. If this is about maintaining ndmhealth - let it say so.  Also new files should come with the header / copyright message.

Reply:
Done